crates/db_schema/src/impls/post.rs

```diff
@@ -27,7 +27,7 @@ use crate::{
   },
 };
 use ::url::Url;
-use chrono::{Duration, Utc};
+use chrono::{TimeDelta, Utc};
```

Comment:
chrono::Duration was removed, and is now `TimeDelta::try_x` . 

It now return an Option, that checks the bound, so this represents an error that needs to be handled.